sympy/calculus/util.py

```diff
@@ -539,6 +581,16 @@ def lcim(numbers):
     The numbers can be rational or irrational or a mixture of both.
     `None` is returned for incommensurable numbers.
 
+    Parameters
+    ==========
+    numbers : int[]
+        Numbers whose lcim has to be founed. They can be rational or irrational or a mixture of both.
```

Comment:
founed -> found

What does `int[]` mean?

Reply:
Sorry, I was just trying to denote an integer array. What other way is to do that?

Reply:
If the docstring is correct and "they can be rational or irrational or a mixture of both" then this isn't an integer array.
```
numbers: list
    Numbers (rational and/or irrational) for which lcim is to be found.
```

Reply:
Hi, thanks, I have fixed it.